viewer/tsconfig.test.json

```
@@ -0,0 +1,35 @@
+{
+  "compilerOptions": {
+    "incremental": true,
+    "target": "ES2019",
+    "module": "esnext",
+    "declaration": true,
+    "declarationDir": "dist",
+    "outDir": "dist-ts",
+    "rootDir": "src",
+    "esModuleInterop": true,
+    "downlevelIteration": true,
+    "sourceMap": true,
+    "moduleResolution": "node",
+    "strict": true,
+    "stripInternal": true,
+    "noUnusedLocals": false,
+    "noUnusedParameters": false,
+    "typeRoots": [
+        "node_modules/@types",
+        "src/@types"
+    ],
+    "lib": [
+        "dom",
+        "esnext"
+    ],
+
+    "baseUrl": "./src/",
+    "paths": {
+      "@/*": ["*"]
+    }
+  },
+  // include is also used for eslint to understand which files should be linted
+  "include": ["*.js", "*.ts", "src/**/*", ".eslintrc.js", "script", "src/@types"],
+  "exclude": ["node_modules/**/*", "dist/**/*"]
+}
```

Let's use [extend](https://www.typescriptlang.org/tsconfig#extends) here.

```suggestion
{
  "extends": "./tsconfig.json",
  "compilerOptions": {
    "noUnusedLocals": false,
    "noUnusedParameters": false
  }
}
```